.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         runs-on: ubuntu-latest
         permissions: write-all
         container:
-            image: public.ecr.aws/u6u3h0g1/docker-nodejs-ci:2.2
+            image: public.ecr.aws/studiographene/ci:node-18-alpine
```

Comment:
Can we update it to node-20? Node 20 is LTS

Reply:
Node image version updated to node-20-alpine